src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.builds

```diff
@@ -1,89 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), packaging.props))\packaging.props" />
```

Comment:
Why the different convention?  Couldn't you have named these dir.props and dir.targets in src/.nuget?